packages/ncids-css/patches/@uswds+uswds+3.3.0.patch

```diff
@@ -5,18 +5,108 @@ diff --git a/node_modules/@uswds/uswds/packages/.DS_Store b/node_modules/@uswds/
 new file mode 100644
 index 0000000..e69de29
 diff --git a/node_modules/@uswds/uswds/packages/usa-collection/src/styles/_usa-collection.scss b/node_modules/@uswds/uswds/packages/usa-collection/src/styles/_usa-collection.scss
-index 7a52382..30f682e 100644
+index 7a52382..0b92765 100644
 --- a/node_modules/@uswds/uswds/packages/usa-collection/src/styles/_usa-collection.scss
 +++ b/node_modules/@uswds/uswds/packages/usa-collection/src/styles/_usa-collection.scss
-@@ -6,7 +6,7 @@ $collection-thumb-margin: 2;
+@@ -1,12 +1,13 @@
+ @use "uswds-core" as *;
+ 
+-$collection-thumb-width: 10;
+-$collection-thumb-width-mobile: 8;
+-$collection-thumb-margin: 2;
++$collection-calendar-width: 10;
++$collection-calendar-width-mobile: 8;
++$collection-calendar-img-margin: 2;
```

Comment:
can you rename this to `collection-calendar-margin` so we don't confuse it with the img block